daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/EvaluationOrderTest.scala

```diff
@@ -875,7 +875,6 @@ class EvaluationOrderTest extends AnyFreeSpec with Matchers with Inside {
           inside(res) { case Success(Right(_)) =>
             msgs shouldBe Seq(
               "starts test",
-              "queries contract",
```

Comment:
For this set of tests, we are removing observations that information has originated from our internal caches.

Do we want to do this, or maybe you're testing caching elsewhere?

Reply:
This string only appears when we ask the ledger for a contract
We do still memoize the answers to NeedContract (in Engine.scala) but it is a pure optimization with no real importance,
And definitely no relevance to evaluation order testing which is being done here.

Reply:
>   And definitely no relevance to evaluation order testing which is being done here.
We still need to know when the evaluation may fail because the ledger return contract non found. 
So I will argue we should better add the string everywhere that removing it.